src/midgard/util.cc

```diff
@@ -697,5 +701,18 @@ polygon_t to_boundary(const std::unordered_set<uint32_t>& region, const Tiles<Po
   return polygon;
 }
 
+std::string encode64(const std::string& val) {
+  using namespace boost::archive::iterators;
+  using It = base64_from_binary<transform_width<std::string::const_iterator, 6, 8>>;
+  auto tmp = std::string(It(std::begin(val)), It(std::end(val)));
+  return tmp.append((3 - val.size() % 3) % 3, '=');
```

Comment:
@mookerji this is where the padding comes from

Reply:
yeah im pretty sure this is where the padding comes from. i think at some point some use-case must have needed to pad to the nearest multiple of 3 bytes but im failing to remember where/why/what...

Reply:
https://stackoverflow.com/questions/4080988/why-does-base64-encoding-require-padding-if-the-input-length-is-not-divisible-by/18518605#18518605

Reply:
it seems like we need to modify the decode function to ignore the padding

Reply:
which i think can be done by removing null characters from the end?

Reply:
Fixed